src/css/organisms/filter-form.scss

```diff
@@ -56,6 +56,28 @@
       padding-bottom: 0px;
     }
   }
+
+  /* - Internet Explorer 10–11
+   - Internet Explorer Mobile 10-11 */
+  :-ms-input-placeholder {
+    font-size: 0.8rem;
+    padding: 4px;
+  }
```

Comment:
I reckon we can gracefully degrade here. Support for `rem` units in those versions is also not great, although it should work with `font-size`.